Lib/test/test_pickle.py

```diff
@@ -700,7 +700,7 @@ def test_multiprocessing_exceptions(self):
 
 
 def load_tests(loader, tests, pattern):
-    tests.addTest(doctest.DocTestSuite())
+    tests.addTest(doctest.DocTestSuite(pickle))
```

Comment:
Oh, this looks like a bugfix. It should be backported. Can you extract this change into a new PR? It's not directly related to CLI changes.

Reply:
Yes, sure, should i create new issue or i send a PR for the current issue?